src/Umbraco.Web/Editors/ImagesController.cs

```diff
@@ -124,53 +122,11 @@ private HttpResponseMessage GetResized(string imagePath, int width, string suffi
                 return Request.CreateResponse(HttpStatusCode.NotFound);
             }
 
-            var thumbFilePath = imagePath.TrimEnd(ext) + "_" + suffix + ext;
+            //redirect to ImageProcessor thumbnail with rnd generated from last write time of original media file
             var fullOrgPath = mediaFileSystem.GetFullPath(mediaFileSystem.GetRelativePath(imagePath));
-            var fullNewPath = mediaFileSystem.GetFullPath(mediaFileSystem.GetRelativePath(thumbFilePath));
-            var thumbIsNew = mediaFileSystem.FileExists(fullNewPath) == false;
-            if (thumbIsNew)
-            {
-                //we need to generate it
-                if (mediaFileSystem.FileExists(fullOrgPath) == false)
-                {
-                    return Request.CreateResponse(HttpStatusCode.NotFound);
-                }
-
-                using (var fileStream = mediaFileSystem.OpenFile(fullOrgPath))
-                {
-                    if (fileStream.CanSeek) fileStream.Seek(0, 0);
-                    using (var originalImage = Image.FromStream(fileStream))
-                    {
-                        //If it is bigger, then do the resize
-                        if (originalImage.Width >= width && originalImage.Height >= width)
-                        {
-                            ImageHelper.GenerateThumbnail(
-                                originalImage,
-                                width,
-                                fullNewPath,
-                                ext.Replace(".", ""),
-                                mediaFileSystem);
-                        }
-                        else
-                        {
-                            //just return the original image
-                            fullNewPath = fullOrgPath;
-                        }
-                        
-                    }
-                }
-            }
-
-            var result = Request.CreateResponse(HttpStatusCode.OK);
-            //NOTE: That we are not closing this stream as the framework will do that for us, if we try it will
-            // fail. See http://stackoverflow.com/questions/9541351/returning-binary-file-from-controller-in-asp-net-web-api
-            var stream = mediaFileSystem.OpenFile(fullNewPath);
-            if (stream.CanSeek) stream.Seek(0, 0);
-            result.Content = new StreamContent(stream);
-            result.Headers.Date = mediaFileSystem.GetLastModified(imagePath);
-            result.Content.Headers.ContentType = new MediaTypeHeaderValue(System.Web.MimeMapping.GetMimeMapping(imagePath));
-            
-            return result;
+            var response = Request.CreateResponse( HttpStatusCode.Found );
+            response.Headers.Location = new Uri( string.Format( "{0}?rnd={1}&width={2}", imagePath, string.Format( "{0:yyyyMMddHHmmss}", System.IO.File.GetLastWriteTime( fullOrgPath ) ), width ), UriKind.Relative );

```

Comment:
If you can push a fix for this, I can test and try to get this in to 7.4.0 which is due for release very early Jan.